content/09.overfitting.md

```diff
@@ -1,33 +1,36 @@
 ## Tip 7: Address deep neural networks' increased tendency to overfit the dataset {#overfitting}
 
-Overfitting is one of the most significant dangers you'll face in deep learning (and traditional machine learning).
-Put simply, overfitting occurs when a model fits patterns in the training data too closely, includes noise or non-scientifically relevant perturbations, or in the most extreme case, simply memorizes patterns in the training set.
+Overfitting is a challenge inherent to machine learning in general, and is one of the most significant challenges you'll face when applying deep learning specifically.
+Overfitting occurs when a model fits patterns in the training data so closely that it is including non-generalizable noise or non-scientifically relevant perturbations in the relationships it is learning.
+In other words, it is as if the model is memorizing patterns in the specific data set it is training on rather than learning general relationships that hold across similar datasets.
```

Comment:
```suggestion
In other words, the model fits patterns that are overly specific to the data it is training on rather than learning general relationships that hold across similar datasets.
```